lib/events.js

```diff
@@ -321,9 +297,9 @@ function enhanceStackTrace(err, own) {
   const ownStack = ArrayPrototypeSlice(
     StringPrototypeSplit(own.stack, '\n'), 1);
 
-  const { 0: len, 1: off } = identicalSequenceRange(ownStack, errStack);
+  const { len, offset } = identicalSequenceRange(ownStack, errStack);
   if (len > 0) {
-    ArrayPrototypeSplice(ownStack, off + 1, len - 2,
+    ArrayPrototypeSplice(ownStack, offset + 1, len - 2,
```

Comment:
These changes seem somewhat unrelated and appear to undo some work around "safe" destructuring.

Reply:
They are, I forgot to do this in a former commit when I moved this part to internal/inspect, since it's required to identify identical stack frames.
If you prefer, I'll separate it in a different commit or PR.

Reply:
Ah, I didn't know that there were two identical implementations of `identicalSequenceRange` with different return types. That's fine then, I guess.